app/core/security.py

```diff
@@ -38,10 +41,10 @@ def create_token_with_id(self, userId):
         """Creates a JWT token with the `id` of the `user`."""
         token = jwt.encode(
             payload={
-                "subject": userId,
-                "expire_at": (datetime.utcnow() + timedelta(minutes=30)).ctime(),
+                "sub": userId,
+                "exp": (datetime.utcnow() + timedelta(minutes=2)).timestamp(),
```

Comment:
Could the 2 be replaced with settings.ACCESS_TOKEN_EXPIRE_MINUTES

Reply:
oops must have missed that, latest commit resolves that